src/SourceBuild/content/repo-projects/scenario-tests.proj

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <BuildCommand>$(StandardSourceBuildCommand) $(StandardSourceBuildArgs)</BuildCommand>
+  </PropertyGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <ItemGroup>
+    <RepositoryReference Include="source-build-externals" />
+    <RepositoryReference Include="source-build-reference-packages" />
+    <RepositoryReference Include="arcade" />
```

Comment:
This in theory should depend on everything - perhaps just declaring a dependency on installer is sufficient.

Reply:
It doesn't actually need to depend on everything. It has no dependencies on the installer or runtime or anything like that for building. Only arcade, SBE and SBRP.

Reply:
The project executions would depend on the installed SDK, but not to build the harness and the xunit code.

Reply:
Right I get that but from a build perspective it could depend on all repos.  You want it to compile against the latest libraries for example.  Declaring a dependency on installer feels maintainable versus declaring all repo dependencies.

Reply:
That's not how the tests work though. The test projects are not supposed to **build** at VMR build time. They build using on an installed product. So for instance, with the template tests, new projects are created using the installed SDK, then built at test execution time.

Reply:
The only dependencies that the **repo** build has are SBE and SBRP, and arcade. There are no other package refs that would be used at scenario-tests's repo build time.

Reply:
Yes, of course - I wasn't thinking about this correctly.